package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "node-js-course",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "prestart": "babel src -d dist",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon ./dist/app.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sasha-klimashevich/node-js-course.git"
+  },
+  "author": "Aliaksandra Klimashevich",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/sasha-klimashevich/node-js-course/issues"
+  },
+  "homepage": "https://github.com/sasha-klimashevich/node-js-course#readme",
+  "dependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.0",
+    "babel-preset-env": "^1.6.1",
```

Comment:
Babel's module should not be in dependencies section. They should be in devDep.
Check what modules should in devDep section, please

FYI: [dependencies-vs-devdependencies](https://medium.com/@stalonadsl948/dependencies-vs-devdependencies-926e096a3dee)

Reply:
Done, thx